woocommerce/woocommerce-android
# [Mobile Payments] Handle WCPay plugin setup not finished with auth webview

Closes: #9685


### Description

The PR adds CTA on the "WCPay setup not finished" onboarding error screen. The CTA opens webview where a user can finish the setup

### Testing instructions

The only way I found our how to simulate this error in the test environment is to deactivate WCPay Dev plugin
* Open More -> Payments -> Click on the error
* Notice new CTA
* Click on it and notice that WCPay onboarding opens in the webview of wp-admin (you should be logged in automatically)

Also, noticed the event being tracked:

```
Tracked: card_present_onboarding_cta_tapped, Properties: {"reason":"plugin_setup_tapped","plugin_slug":"woocommerce-payments","country":"US","card_reader_model":"STRIPE_M2","blog_id":198219640,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/anotherpaymentssite.wpcomstaging.com"}
```

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4923871/a6416a85-a100-4e34-9fdc-edb533601db5
